Makefile

```diff
@@ -210,17 +219,19 @@ test-ci-native: | test/addons/.buildstamp
 		$(TEST_CI_ARGS) $(CI_NATIVE_SUITES)
 
 # This target should not use a native compiler at all
-test-ci-js:
+test-ci-js: | clear-stalled
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
 		$(TEST_CI_ARGS) $(CI_JS_SUITES)
+	! ( ps awwx | grep Release/node | grep -v grep )
```

Comment:
Would it be better to print out the info on what's still running here and then cleanup.  This will leave them running until the next ci run right ?